app/src/main/java/com/realityexpander/tasky/agenda_feature/common/RepositoryResult.kt

```diff
@@ -1,6 +1,15 @@
 package com.realityexpander.tasky.agenda_feature.common
```

Comment:
Instead of using a `common` package inside a feature, you'll almost always be able to clearly put a class in a certain package. This `RepositoryResult` will fit well into domain. If it's some kind of util function for the UI, it should go in `feature->presentation->util` for example and same for data. 
In the end, the different layers are there for a reason and putting stuff into a separate `common` package will completely separate it from any layer.

Reply:
OK sounds good. Still a bit confused about what goes where, and starting to understand.